drink.rb

```diff
@@ -0,0 +1,12 @@
+class Drink
+	attr_accessor :name, :description
```

Comment:
Lots of great improvements!

Something about the spacing is still off; check and see what your text editor is set to.